### PR TITLE
feat(locale): update Korean color, date, location, word, company

### PR DESCRIPTION
**Pull request by thilllon**
I added Korean color.human, date.month, date.weekday, location.secondary_address, word.adjective, word.adverb, word.noun and extended Korean company.suffix.

For Koreans reading this, I know that adjectives and adverbs cannot be paired with English, grammatically strictly speaking. But I tried to choose words that made the most sense. 👍

### CODE REVIEW

File: src/locales/ko/date/month.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
+// Source: http://unicode.org/cldr/trac/browser/tags/release-27/common/main/en.xml#L1799
```
Comment (ST-DDT) [inline src/locales/ko/date/month.ts]: This link doesn't work.

File: src/locales/ko/date/weekday.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+// Source: http://unicode.org/cldr/trac/browser/tags/release-27/common/main/en.xml#L1847
```
Comment (ST-DDT) [inline src/locales/ko/date/weekday.ts]: This link doesn't work.

Comment (thilllon) [reply src/locales/ko/date/weekday.ts]: Sorry, my bad. I deleted it.

File: src/locales/ko/date/month.ts (hunk 1 above)
Comment (thilllon) [reply src/locales/ko/date/month.ts]: I deleted it.

**Review by ST-DDT — DISMISSED** [review]

**Comment by ST-DDT** [comment]
The values look good to me (checked via Google Translate), but there are some duplicates.
Please remove the duplicates.

File: src/locales/ko/color/human.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  '올리브',
+  '은색',
+  '자두색',
+  '자주색',
+  '적갈색',
+  '주황색',
+  '청록색',
+  '초록색',
+  '푸른색',
+  '하늘색',
```
Comment (thilllon) [inline src/locales/ko/color/human.ts:24]: duplicated

File: src/locales/ko/date/month.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    '4월',
+    '5월',
+    '6월',
+    '7월',
+    '8월',
+    '9월',
+    '10월',
+    '11월',
+    '12월',
+  ],
```
Comment (thilllon) [inline src/locales/ko/date/month.ts]: wide_context is optional and same as wide.

File: src/locales/ko/date/month.ts [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
```
Comment (thilllon) [inline src/locales/ko/date/month.ts]: abbr_context is optional and same as wide.

File: src/locales/ko/word/adverb.ts [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  '분명히',
+  '비정상적으로',
+  '빠르게',
+  '심하게',
+  '아름답게',
+  '어색하게',
+  '언제나',
+  '열심히',
+  '용감하게',
+  '용기 있게',
```
Comment (thilllon) [inline src/locales/ko/word/adverb.ts]: duplicated

File: src/locales/ko/word/noun.ts [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+  '비행사',
+  '비행선',
+  '비행장',
+  '성인',
+  '성취',
+  '성취자',
+  '쇠붙이',
+  '술',
+  '승인',
+  '신천옹',
```
Comment (thilllon) [inline src/locales/ko/word/noun.ts]: remove poorly used word

File: src/locales/ko/word/noun.ts [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+  '아드레날린',
+  '아마존',
+  '아몬드',
+  '악어',
+  '안건',
+  '알더',
+  '알리다',
+  '알파벳',
+  '알파카',
+  '알팔파',
```
Comment (thilllon) [inline src/locales/ko/word/noun.ts]: remove poorly used word

File: src/locales/ko/word/noun.ts [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+  '쇠붙이',
+  '술',
+  '승인',
+  '신천옹',
+  '아드레날린',
+  '아마존',
+  '아몬드',
+  '악어',
+  '안건',
+  '알더',
```
Comment (thilllon) [inline src/locales/ko/word/noun.ts]: remove poorly used word

File: src/locales/ko/word/noun.ts [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+  '알파벳',
+  '알파카',
+  '알팔파',
+  '애프터셰이브',
+  '앨범',
+  '양',
+  '양자',
+  '어댑터',
+  '에어백',
+  '에어버스',
```
Comment (thilllon) [inline src/locales/ko/word/noun.ts]: remove poorly used word

File: src/locales/ko/word/noun.ts [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+  '옹호',
+  '외계인',
+  '용돈',
+  '유연',
+  '유추',
+  '음향학',
+  '응집',
+  '이점',
+  '일',
+  '장백의',
```
Comment (thilllon) [inline src/locales/ko/word/noun.ts]: remove poorly used word

File: src/locales/ko/word/noun.ts [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+  '술',
+  '승인',
+  '신천옹',
+  '아드레날린',
+  '아마존',
+  '아몬드',
+  '악어',
+  '안건',
+  '알더',
+  '알리다',
```
Comment (thilllon) [inline src/locales/ko/word/noun.ts]: is not a noun

File: src/locales/ko/color/human.ts [hunk 13]
```diff
@@ -0,0 +1,28 @@
...
+  '하늘색',
```
Comment (thilllon) [inline src/locales/ko/color/human.ts]: duplicated

**Comment by thilllon** [comment]
> The values look good to me, but there are some duplicates. Please remove the duplicates.

I fixed. Take a look please.

**Comment by ST-DDT** [comment]
> > The values look good to me, but there are some duplicates. Please remove the duplicates.
> 
> I fixed. Take a look please.

Awesome. Thanks for your contribution.

**Review by ST-DDT — DISMISSED** [review]

**Review by ejcheng — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Pull request merged**